builtin_interfaces/Time stamp

# Usually only one is filled
UniqueTopicFlowStat[] stats
UniqueTopicFlowTunnelStat[] tunnel_stats 
